Production Release

shrunyan opened the pull request.

japhethLG committed:
revert: Revert to old pricing page (#2459)

# Description

Revert to old pricing page

## Type of change

Please delete options that are not relevant.

- [x] Revert feature (non-breaking change which reverts functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce.

- [x] Manual Test

# Screenshots / Screen recording

![image](https://github.com/zesty-io/website/assets/83058948/ab5a897a-282b-4b32-a82e-0f77d84da9bb)

glespinosa committed:
fix(user & domain): fixing invalid date (#2465)

fixing invalid date on users and domains tab

Will close : https://github.com/zesty-io/manager-ui/issues/2716

japhethLG committed:
fix: Zesty.io Demo Page spacing edits (#2468)

# Description

Fixed the whitespace in demo form

Fixes #2467 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce.

- [x] Manual Test
- [ ] Unit Test
- [ ] E2E Test

# Screenshots / Screen recording

Please add screenshots or recording if applicable
Before: 

![image](https://github.com/zesty-io/website/assets/83058948/937f9c4b-fc71-4b24-a9fd-69bd33ee2246)

After:

![image](https://github.com/zesty-io/website/assets/83058948/1045d539-5e81-4080-b60d-c0f6038438ce)

japhethLG committed:
fix: Zesty.io Mindshare links falling back to old content/not displaying properly (#2470)

# Description

Fixed Zesty.io Mindshare links falling back to old content/not
displaying properly

Fixes #2463

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce.

- [x] Manual Test
- [ ] Unit Test
- [ ] E2E Test

# Screenshots / Screen recording
Before:

![image](https://github.com/zesty-io/website/assets/83058948/a86c3600-4c11-4a84-a665-775085c253d5)

After:

![image](https://github.com/zesty-io/website/assets/83058948/1ae206ce-91cc-4500-b5d2-984d513c3c09)

japhethLG committed:
fix: Error on Homepage Demo CTA (#2474)

# Description

Fix wrong props of homepage cta; Removed spans in headings for articles

Fixes #2472, #2473

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce.

- [x] Manual Test

# Screenshots / Screen recording
Before:

![image](https://github.com/user-attachments/assets/ef1994f5-81ee-42cd-b185-a4d166c27417)

![image](https://github.com/user-attachments/assets/f49dd7ff-8872-4adf-a1c7-b866c280a9e2)

After:

![image](https://github.com/user-attachments/assets/416038be-33cd-4e60-8a58-1ec2a37fa71d)

![image](https://github.com/user-attachments/assets/0c761b0e-a65c-46cc-aeec-66182d8f5ae3)

agalin920 committed:
feat: Add new system roles (#2475)

Adds new system roles: Developer Contributor, Access Admin

PR for adding them to legacy accounts:
https://github.com/zesty-io/accounts-ui/pull/248

japhethLG committed:
fix: Announcement is not reflecting the latest announcement (#2477)

# Description

Add a sort logic for product announcements. Remove "Good Night" when the
user logs in at night.

Fixes #2476 [#250](https://github.com/zesty-io/accounts-ui/issues/250)
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] Manual Test

# Screenshots / Screen recording

Please add screenshots or recording if applicable
Before:

![image](https://github.com/user-attachments/assets/ca977a94-786b-4219-9522-4ecddbbac870)

After:

![image](https://github.com/user-attachments/assets/a76aeac6-8961-4e56-9b16-bba34f2a968a)

japhethLG committed:
feat: docs redirects (#2488)

# Description

Redirects zesty docs to readme

Fixes #2487 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce.

- [x] Manual Test

# Screenshots / Screen recording




https://github.com/user-attachments/assets/05a4fb40-231b-41e1-9a5c-46288f1b06d2

---------

Signed-off-by: Japheth Louie M. Gofredo <83058948+japhethLG@users.noreply.github.com>

japhethLG committed:
feat: docs redirects (#2490)

# Description

This is a fix for the previous redirects of zesty docs to readme. Also
added small fix for removing acorns logo

Fixes #2487 #2484 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce.

- [x] Manual Test

# Screenshots / Screen recording




https://github.com/user-attachments/assets/05a4fb40-231b-41e1-9a5c-46288f1b06d2

---------

Signed-off-by: Japheth Louie M. Gofredo <83058948+japhethLG@users.noreply.github.com>

japhethLG committed:
fix: Remove Acorn, BJs and Cornershop logos from all components in the marketing website (#2491)

# Description

Removed Acorn, BJs and Cornershop logos from all components in the
marketing website (Homepage, Demo, Insurance CMS).

Fixes #2484 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] Manual Test

# Screenshots / Screen recording


![image](https://github.com/user-attachments/assets/6ac7e196-bc3a-4d01-afec-a364f66e1e59)

![image](https://github.com/user-attachments/assets/ee5b6d8c-3c75-466b-93b1-64f4f8b98d8d)

![image](https://github.com/user-attachments/assets/33221a89-a95a-4598-80d9-1b48288bc56a)

finnar-bin committed:
feat: Update homepage logos (#2486)

# Description
Update homepage company logos

Resolves #2483 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Manual Test

# Screenshots / Screen recording

![image](https://github.com/user-attachments/assets/36e5031b-da92-446a-b198-080181fc5387)

---------

Signed-off-by: Stuart Runyan <stuart@zesty.io>
Co-authored-by: Stuart Runyan <stuart@zesty.io>

finnar-bin committed:
feat: Custom Roles (#2481)

# Description
Added a page to manage an instance's custom roles.
Closes #2479 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Dependencies
- [ ] https://github.com/zesty-io/fetch-wrapper/pull/66
- [x] https://github.com/zesty-io/material/pull/109
- [ ] https://github.com/zesty-io/accounts-api/pull/285
- [ ] https://github.com/zesty-io/base-api/pull/207

# How Has This Been Tested?
- [x] Manual Test


# Screenshots / Screen recording

[screen-recorder-tue-oct-29-2024-15-02-58.webm](https://github.com/user-attachments/assets/e75765ef-8ab1-4936-b925-55f39977591a)

---------

Co-authored-by: Stuart Runyan <stuart@zesty.io>

finnar-bin committed:
fix: Open permissions tab upon successful creation of custom role (#2496)

# Description

Automatically opens the permissions tab upon successful creation of a
new custom role


## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce.
- [x] Manual Test

# Screenshots / Screen recording

[screen-recorder-tue-dec-10-2024-09-28-21.webm](https://github.com/user-attachments/assets/95d107ee-a993-4824-b553-1fce444c8cdc)

zcolah commented:
# Bugs

## High Priority

## Medium Priority

## Low Priority

# Figma VQA

- [ ]

shrunyan committed:
resolve stage to production conflict

finnar-bin committed:
feat: resolve vqa comments for custom roles (#2499)

# Description
Resolve vqa comments for custom roles
Requires https://github.com/zesty-io/material/pull/111

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
- [x] Manual Test
- [ ] Unit Test
- [ ] E2E Test

---------

Co-authored-by: shrunyan <shrunyan@gmail.com>
